### ISSUE TITLE
Pug transformer filters not working

### ISSUE THREAD

**Issue by BloodSucker123** [pugjs/pug#3034]
I tried to use the custom filters and its working but the transformer filters are not.
I follow the documentation and it says `npm install --save jstransformer-coffee-script` and I then the transformer should work as stated `Now, you should be able to render the following template`.
When I tried the example code which is
`script
  :coffee-script
    console.log 'This is coffee script'`
but it does do nothing on the rendered template instead its not being filtered.

Can someone help me. how to use the transformer filter? or did I miss something in order to make it work?
I'm using pug-cli to render/compile my templates

**Comment by ForbesLindesay** [comment]
Please can you provide a step-by-step description of what you tried. What packages did you install, which commands did you use to install them, what does the pug code that's not working looks like.

**Comment by BloodSucker123** [comment]
Sorry for this issue. I finally found out why JSTransformer is not working. It is due to that I'm using a pug-cli which is not part of the pugjs renderer. I already found a solution which is I have to load my configuration of pug-cli which contains options that has filters. In this case I have to write and require the JSTransformer that I will use.

`var uglify = require('jstransformer')(require('jstransformer-uglify-js'));`

`module.exports={`
`    filters:{`
`        'uglify-js': function filterFunction (str, opts) {`
`	  	return uglify.render(str,opts).body;`
`        }`
`    }`
`}`
